src/packages/engine-core/src/NAPIEngine.ts

```diff
@@ -41,6 +41,7 @@ type QueryEngineConfig = {
   datasourceOverrides?: Record<string, string>
   logLevel: QueryEngineLogLevel
   telemetry?: QueryEngineTelemetry
+  configDir?: string
```

Comment:
Actually, it seems it's unused? Can't find anything related to `configDir`. So we can remove?

Reply:
L269

Reply:
It is definitely used and needed

Reply:
I saw this indeed `configDir: this.config.cwd,` but I couldn't find where it's actually used 🤔

Reply:
Seems unused, passed as extra

Reply:
Some [context](https://github.com/prisma/prisma-engines/blob/master/query-engine/query-engine-napi/src/engine.rs#L96) from the Rust side

Reply:
It's actually required in engine, thanks Will for the link https://github.com/prisma/prisma-engines/blob/master/query-engine/query-engine-napi/src/engine.rs